src/Context/Context.tsx

```diff
@@ -12,25 +13,46 @@ export const AuthContext = createContext<AuthData | any>(null);
 
 export default function AuthContextProvider(props: any) {
   const [loginData, setLoginData] = useState(null);
+  const [userName, setUserName] = useState(null);
+  const [profileImage, setProfileImage] = useState(null);
   const saveLoginData = () => {
     const encodedToken: any = localStorage.getItem("token");
     const decodedToken: any = jwtDecode(encodedToken);
+
+    if (loginData) {
+      getCurrentUser();
+    }
     setLoginData(decodedToken);
   };
+
+  const getCurrentUser = () => {
+    axiosInstance
+      .get(AUTH_URLS.get_current_user(loginData?._id))
```

Comment:
We need to provide the type to the get method to make sure that we are accessing the right values.